samples/ios/app/glean-sample-app/ViewController.swift

```diff
@@ -53,4 +55,14 @@ class ViewController: UIViewController {
         Custom.counter.add()
         Pings.shared.sample.send()
     }
+
+    @IBAction func enableToggled(_: Any) {
```

Comment:
Even though this is an example, we probably should still set the initial value of the switch to the current `getUploadEnabled` in the `viewDidLoad()` function.